src/ci/docker/host-x86_64/x86_64-gnu-aux/Dockerfile

```diff
@@ -28,3 +28,5 @@ RUN /scripts/cmake.sh
 
 ENV RUST_CONFIGURE_ARGS --build=x86_64-unknown-linux-gnu
 ENV RUST_CHECK_TARGET check-aux
+
+ENV SCRIPT python3 ../x.py test compiler --stage 2
```

Comment:
It also seems suspicious to have a check target and this - I don't think that will work: https://github.com/rust-lang/rust/blob/f9cba63746d0fff816250b2ba7b706b5d4dcf000/src/ci/run.sh#L157

Reply:
Thanks for the review.
I will change it so that the test is run by specifying each compiler/rustc_*.

Reply:
Is this too naive?
```suggestion
ENV SCRIPT python3 x.py test compiler/rustc_apfloat compiler/rustc_arena compiler/rustc_ast compiler/rustc_ast_lowering compiler/rustc_ast_passes compiler/rustc_ast_pretty compiler/rustc_attr compiler/rustc_borrowck compiler/rustc_builtin_macros compiler/rustc_codegen_llvm compiler/rustc_codegen_ssa compiler/rustc_const_eval compiler/rustc_data_structures compiler/rustc_driver compiler/rustc_error_codes compiler/rustc_error_messages compiler/rustc_errors compiler/rustc_expand compiler/rustc_feature compiler/rustc_fs_util compiler/rustc_graphviz compiler/rustc_hir compiler/rustc_hir_pretty compiler/rustc_incremental compiler/rustc_index compiler/rustc_infer compiler/rustc_interface compiler/rustc_lexer compiler/rustc_lint compiler/rustc_lint_defs compiler/rustc_llvm compiler/rustc_log compiler/rustc_macros compiler/rustc_metadata compiler/rustc_middle compiler/rustc_mir_build compiler/rustc_mir_dataflow compiler/rustc_mir_transform compiler/rustc_monomorphize compiler/rustc_parse compiler/rustc_parse_format compiler/rustc_passes compiler/rustc_plugin_impl compiler/rustc_privacy compiler/rustc_query_impl compiler/rustc_query_system compiler/rustc_resolve compiler/rustc_save_analysis compiler/rustc_serialize compiler/rustc_session compiler/rustc_smir compiler/rustc_span compiler/rustc_symbol_mangling compiler/rustc_target compiler/rustc_trait_selection compiler/rustc_traits compiler/rustc_ty_utils compiler/rustc_type_ir compiler/rustc_typeck --stage 2
```

Reply:
That looks good to me :) Sorry for the delay, this got lost in my notifications. FYI you can use `@rustbot ready` to mark this as waiting on my review.

I am slightly concerned that this won't pick up on newly added crates. I don't see a way to avoid that without adding some `x.py test compiler-unit-tests` alias, which @Mark-Simulacrum had concerns about in the past. IIRC you were worried that this isn't really a useful thing to run except for CI? but I think it would be better to give a warning and still allow you to do it rather than have newly added crates not get unit tests run.

Anyway, this is fine for now and we can consider improving it later.